docs/tools/vdb_table/data/anariai.json

```diff
@@ -152,24 +152,28 @@
     "value": 0,
     "unlimited": false,
     "source_url": "https://github.com/Anari-AI",
-    "comment": ""
+    "comment": "",
```

Comment:
https://github.com/Anari-AI this is not the right github repo (it is an org handle). Let's add the ability to mark as not applicable in case of closed source projects

Reply:
I am changing it to keep source_url empty in case of closed source projects.